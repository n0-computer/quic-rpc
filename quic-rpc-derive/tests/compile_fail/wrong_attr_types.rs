use quic_rpc_derive::rpc_requests;

#[rpc_requests(Service)]
enum Enum {
    #[rpc(fnord = Bla)]
    A(u8),
}

fn main() {}