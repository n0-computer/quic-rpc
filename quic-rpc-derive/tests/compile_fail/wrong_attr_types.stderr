error: rpc requires a response type
 --> tests/compile_fail/wrong_attr_types.rs:5:5
  |
5 |     #[rpc(fnord = Bla)]
  |     ^
